### ISSUE TITLE
Tooling keeps inserting ItemGroup.Compile items into csproj after update to Windows 10 Redstone 3 Preview

### ISSUE THREAD

**Issue by ChristopherHaws** [dotnet/project-system#2810]
Hello,

I updated my computer to the latest Windows 10 Redstone 3 Insider Preview a few days ago, and ever since that update I have been having tooling issues in Visual Studio with .NET Core SDK. Whenever I try to rename a file in Visual Studio, instead of executing a move command, it is creating a copy of the old file but with the new name. It is also adding an item group into my csproj file.

**Steps to reproduce:**
1) Create a new .NET Standard 2.0 class library using Visual Studio
2) Open `Class1.cs` and save it (this will cause the file icon to change to not found):
![image](https://user-images.githubusercontent.com/5934505/30174513-36b984e0-93b0-11e7-9618-7d34f3a91da8.png)

3) Rename `Class1.cs` to `Class2.cs`

Notice that on disk the following files now exist:
```
Class1.cs
Class2.cs
ClassLibraryX.csproj
```

Also notice that your csproj now looks like this (for some reason, it created an item group and added the renamed file):
```xml
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <TargetFramework>netstandard2.0</TargetFramework>
  </PropertyGroup>

  <ItemGroup>
    <Compile Include="Class2.cs" />
  </ItemGroup>
</Project>
```

And when you compile, you will get the following error:
```
Severity	Code	Description	Project	File	Line	Suppression State
Error		Duplicate 'Compile' items were included. The .NET SDK includes 'Compile' items from your project directory by default. You can either remove these items from your project file, or set the 'EnableDefaultCompileItems' property to 'false' if you want to explicitly include them in your project file. For more information, see https://aka.ms/sdkimplicititems. The duplicate items were: 'Class2.cs'	ClassLibraryX	C:\Program Files\dotnet\sdk\2.0.0\Sdks\Microsoft.NET.Sdk\build\Microsoft.NET.Sdk.DefaultItems.targets	263	

```

When I debug the project-system project, I get the following exception:
```
CPS Error: 0 : Exception while reading file attributes for "C:\Users\chaws\Documents\Visual Studio 2017\Projects\ClassLibrary6\ClassLibrary6\Class1.cs~RF3ac9ae.TMP".  System.IO.FileNotFoundException: Could not find file 'C:\Users\chaws\Documents\Visual Studio 2017\Projects\ClassLibrary6\ClassLibrary6\Class1.cs~RF3ac9ae.TMP'.
File name: 'C:\Users\chaws\Documents\Visual Studio 2017\Projects\ClassLibrary6\ClassLibrary6\Class1.cs~RF3ac9ae.TMP'
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.File.GetAttributes(String path)
   at Microsoft.VisualStudio.ProjectSystem.Designers.CoreDirectoryTreeProvider.IsHiddenFile(String fullPath)
CPS Error: 0 : Exception while reading file attributes for "C:\Users\chaws\Documents\Visual Studio 2017\Projects\ClassLibrary6\ClassLibrary6\Class1.cs~RF3ac9ae.TMP".  System.IO.FileNotFoundException: Could not find file 'C:\Users\chaws\Documents\Visual Studio 2017\Projects\ClassLibrary6\ClassLibrary6\Class1.cs~RF3ac9ae.TMP'.
File name: 'C:\Users\chaws\Documents\Visual Studio 2017\Projects\ClassLibrary6\ClassLibrary6\Class1.cs~RF3ac9ae.TMP'
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.File.GetAttributes(String path)
   at Microsoft.VisualStudio.ProjectSystem.Designers.CoreDirectoryTreeProvider.IsHiddenFile(String fullPath)
```

Windows 10 Version:
10.0.16278.1000 (rs3_release.170825-1441)

Visual Studio Version:
Enterprise 2017 v15.4.0 Preview 1.0

**Comment by ChristopherHaws** [comment]
This seems like it could be related to https://github.com/dotnet/project-system/issues/1550 but it is referencing a private VSTS work item that I can't view.

**Comment by davkean** [comment]
Thanks for reporting. This is a dupe of https://developercommunity.visualstudio.com/content/problem/80855/solution-shows-missing-file-icon.html and will be fixed in 15.4 Preview 2.

**Comment by kiwiant** [comment]
Part of it may be a duplicate of https://developercommunity.visualstudio.com/content/problem/80855/solution-shows-missing-file-icon.html however on VS Enterprise 15.3.5 RTW running on Windows 10 Enterprise 16299 I'm still having a build error after adding a new js file and renaming it. This has been happening for the last few releases.

"Duplicate 'Content' items were included. The .NET SDK includes 'Content' items from your project directory by default. You can either remove these items from your project file, or set the 'EnableDefaultContentItems' property to 'false' if you want to explicitly include them in your project file."

I'm reluctant to create a new issue for the same symptoms but let me know if this is required.

Solution is connected to TFS.

When renaming inline.js to forms.js in Solution Explorer:
1. The original file remains in the file system but no longer shows in Solution Explorer
2. A copy of the file with the new name appears in the file system and Solution Explorer
3. .csproj file is updated with:
`  <ItemGroup>
    <Content Include="wwwroot\js\forms.js" />
  </ItemGroup>
`
